Files.Package/Package.appxmanifest

```diff
@@ -68,6 +68,9 @@
         <rescap:Capability Name="allowElevation"/>
         <uap:Capability Name="removableStorage" />
         <Capability Name="internetClient"/>
+        <uap:Capability Name="picturesLibrary"/>
+        <uap:Capability Name="videosLibrary"/>
+        <uap:Capability Name="musicLibrary"/>
```

Comment:
Are this capabilities necessary since we have the broadFileSystemAccess?

Reply:
Since `broadFileSystemAccess` isn't supported on all platforms, I added those permissions in case we're targeting platforms like xbox.

Reply:
While we'd like to, we can't target Xbox since we use a FullTrustProcess so I think the capabilities can be removed.

Reply:
Removed.